BUILDING.md

```diff
@@ -0,0 +1,99 @@
+## Building
+
+You need to set PICO_SDK_PATH in the environment, or pass it to cmake with `-DPICO_SDK_PATH=/path/to/pico-sdk`. To use features such as signing or hashing, you will need to make sure the mbedtls submodule in the SDK is checked out - this can be done by running this from your SDK directory.
+
+```console
+git submodule update --init lib/mbedtls
+```
+
+You also need to install `libusb-1.0` if you want to use the USB functionality.
+
+### Linux / macOS
+
+Use your favorite package tool to install dependencies. For example, on Ubuntu:
+
+```console
+sudo apt install build-essential pkg-config libusb-1.0-0-dev cmake
```

Comment:
Is there a simple `brew` command to install the needed dependencies on macOS?

Reply:
Hmmm, in that case perhaps the "macOS" instructions should be broken out into their own section?

Reply:
Maybe a separate PR to improve (and test) the MacOS instructions once this one has been merged in? This was just copied directly from the existing README